rele/subscription.py

```diff
@@ -53,6 +56,17 @@ def __str__(self):
     def _filter_returns_false(self, kwargs):
         return self._filter_by and not self._filter_by(kwargs)
 
+    @property
+    def scheduler(self):
```

Comment:
From an abstraction point of view I think a `Subscription` doesn't have a scheduler. IMO it would be better to let `Subscriber.consume` deal with the initialization of the scheduler based on the desired number of threads you want to use to consume